standards.py

```diff
@@ -27,48 +25,20 @@
 args = parser.parse_args()
 
 
-hook_path = "./.git/hooks/pre-push"
-hook_exec = f"python3 {os.path.basename(sys.argv[0])} --check"
-
-if args.hook:
-    print("hook: execute")
-
-    if os.path.exists(hook_path):
-        print(f"hook: git pre-push hook file already exists: `{hook_path}`")
-        print("hook: please make sure to backup and delete the existing pre-push hook file")
-        exit("hook: failed")
-
-    print("hook: write git pre-push hook file contents")
-    hook_file = open(hook_path, "w")
-    hook_file.write(f"#!/bin/sh\n\n{hook_exec}\n")
-    hook_file.close()
-
-    print("hook: make git pre-push hook file executable")
-    hook_stat = os.stat(hook_path)
-    os.chmod(hook_path, hook_stat.st_mode | stat.S_IEXEC)
-
-    print("hook: succeeded")
-
-
-if args.unhook:
-    print("unhook: execute")
-
-    hook_path = "./.git/hooks/pre-push"
-    if os.path.isfile(hook_path):
-        print(f"unhook: found file -> `{hook_path}`")
-        delete = False
-        hook_file = open(hook_path, "r")
-        if hook_exec in hook_file.read():
-            delete = True
-        else:
-            print("unhook: existing git pre-push hook file was not created by this script")
-            exit("unhook: failed")
-        hook_file.close()
-        if delete:
-            os.remove(hook_path)
-            print(f"unhook: delete file -> `{hook_path}`")
+tool_min_ver = 6
+ver_run = subprocess.run([args.tool_path, "--version"], capture_output=True)
```

Comment:
apparently, `capture_output` param is available after python 3.7+
`desktop/logging-testing-linux:stable` is on python 3.6.9 which causes an error here.
we should also update python version to at least 3.7 too